src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
+     *
      * @param email - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService currentuser = new UserService();
+
     public boolean login(String email, String password) {
+        if (currentuser.findByEmail(email) != null) {
+            User user = new User(currentuser.findByEmail(email).getEmail(),
+                    currentuser.findByEmail(email).getPassword());
+            if (user.getPassword().equals(password) && user.getPassword().equals(password)) {
```

Comment:
In this line you compare same in left and right side! Here you can check user for null and after this compare his password (you can write this in one line with return for simplifying code)

Reply:
And after this you don`t need write "return true" and "return false"